src/components/Profile.jsx

```diff
@@ -17,4 +18,15 @@ class Profile extends React.Component {
 	}
 }
 
+Profile.propTypes = {
+	user: propTypes.shape({
+		firstName: propTypes.string,
+		lastName: propTypes.string,
+		images: propTypes.arrayOf(propTypes.shape({
+			small: propTypes.string,
+			large: propTypes.string,
+		}))
```

Comment:
This array would also be required, the way it's currently written. Otherwise accessing `[0]` would cause an error!